doc/whats_new.rst

```diff
@@ -135,7 +136,7 @@ Changelog
 Bug
 ~~~
 
-- Fixed bug in :func:`mne_bids.datasets.fetch_faces_data` where downloading multiple subjects was impossible, by `Stefan Appelhoff`_ (`#262 <https://github.com/mne-tools/mne-bids/pull/262>`_)
+- Fixed bug in ``mne_bids.datasets.fetch_faces_data`` where downloading multiple subjects was impossible, by `Stefan Appelhoff`_ (`#262 <https://github.com/mne-tools/mne-bids/pull/262>`_)
```

Comment:
why the change to :func: ?

Reply:
the func needs to go, because it's no longer a func and thus the link would break. This is how Eric told me to do it in MNE-Python, so I just did the same here

Reply:
okay I see, thanks @sappelhoff